pymcp/transport/subscriptions.py

```
@@ -0,0 +1,52 @@
+        "jsonrpc": "2.0",
+        "method": "notifications/subscriptions/acknowledged",
+        "params": {"notifications": notifications},
+    }
+    if isinstance(rpc_id, (str, int)) and not isinstance(rpc_id, bool):
+        ack["params"]["_meta"] = {"io.modelcontextprotocol/subscriptionId": rpc_id}
+    yield format_sse(json.dumps(ack), event="message")
+    try:
+        while not shutdown.is_set() and not await request.is_disconnected():
+            yield ": ping\n\n"
+            await wait_queue_message(asyncio.Queue(), shutdown=shutdown, timeout=heartbeat_interval)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)